src/main/resources/redisson.yml

```
@@ -0,0 +1,13 @@
+  idleConnectionTimeout: 10000
+  connectTimeout: 10000
+  timeout: 3000
+  retryAttempts: 3
+  retryInterval: 1500
+  nodeAddresses:
+    - "redis://localhost:7001"
+    - "redis://localhost:7002"
+    - "redis://localhost:7003"
+    - "redis://localhost:7004"
+    - "redis://localhost:7005"
+    - "redis://localhost:7006"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)